lib/puppet/ssl/ssl_context.rb

```diff
@@ -0,0 +1,27 @@
+require 'puppet/x509'
```

Comment:
I don't understand while this is required here (or in ssl_provider).

Reply:
derp, updated to `puppet/ssl`.